config/storage.yml

```diff
@@ -0,0 +1,34 @@
+test:
```

Comment:
why are we adding the storage and updating the DB schema?

Reply:
I have removed the active storage stuff.